### PR TITLE
RELATED: RAIL-990 Table totals in AD

### PR DESCRIPTION
**Pull request by Hurtak**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2948/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2319/ : SUCCESS in 1m 53s

**Comment by tomasreichmann** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/634/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2320/ : SUCCESS in 3m 03s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1215/ : SUCCESS in 21m 01s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/608/ : FAILURE in 24m 55s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/635/ : SUCCESS in 1m 06s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2322/ : SUCCESS in 2m 35s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1216/ : SUCCESS in 19m 04s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/609/ : SUCCESS in 7m 46s

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,11 +17,14 @@ The REST API versions in the table are just for your information as the values a
 
 ## 6.3.0
 
+### Added
+
+-   Pivot table totals can be turned on with menu. (RAIL-990)
```
Comment (petrjanu) [inline CHANGELOG.md]: 👍

**Pull request merged**